src/PSAppDeployToolkit/Public/Set-ADTIniSection.ps1

```diff
@@ -0,0 +1,151 @@
+﻿#-----------------------------------------------------------------------------
+#
+# MARK: Get-ADTIniSection
+#
+#-----------------------------------------------------------------------------
+
+function Set-ADTIniSection
+{
+    <#
+    .SYNOPSIS
+        Opens an INI file and sets the values of the specified section.
+
+    .DESCRIPTION
+        Opens an INI file and sets the values of the specified section.
+
+    .PARAMETER FilePath
+        Path to the INI file.
+
+    .PARAMETER Section
+        Section within the INI file.
+
+	.PARAMETER Content
+		A hashtable or dictionary object containing the key-value pairs to set in the specified section.
+
+    .PARAMETER Force
+        Specifies whether the INI file should be created if it does not already exist.
+
+    .INPUTS
+        None
+
+        You cannot pipe objects to this function.
+
+    .OUTPUTS
+        None
+
+        This function does not return any output.
+
+    .EXAMPLE
+        Set-ADTIniSection -FilePath "$env:ProgramFilesX86\IBM\Notes\notes.ini" -Section 'Notes' -Content @{'KeyFileName' = 'MyFile.ID'}
+
+        Sets the 'Notes' section to only contain the content specified, supplied as hashtable.
+
+    .EXAMPLE
+        Set-ADTIniSection -FilePath "$env:ProgramFilesX86\IBM\Notes\notes.ini" -Section 'Notes' -Content ([ordered]@{'KeyFileName' = 'MyFile.ID'; 'KeyFileType' = 'ID'})
+
+        Sets the 'Notes' section to only contain the content specified, supplied as an ordered hashtable to maintain the desired order.
+
+    .EXAMPLE
+        Set-ADTIniSection -FilePath "$env:ProgramFilesX86\IBM\Notes\notes.ini" -Section 'Notes' -Content @{}
+
+       Sets the 'Notes' section to be empty by sending an empty hashtable.
+
+    .EXAMPLE
+        Set-ADTIniSection -FilePath "$env:ProgramFilesX86\IBM\Notes\notes.ini" -Section 'Notes' -Content $null
+
+       Sets the 'Notes' section to be empty by sending a null value.
+
+    .NOTES
+        An active ADT session is NOT required to use this function.
+
+        Tags: psadt<br />
+        Website: https://psappdeploytoolkit.com<br />
+        Copyright: (C) 2025 PSAppDeployToolkit Team (Sean Lillis, Dan Cunningham, Muhammad Mashwani, Mitch Richters, Dan Gough).<br />
+        License: https://opensource.org/license/lgpl-3-0
+
+    .LINK
+        https://psappdeploytoolkit.com/docs/reference/functions/Set-ADTIniSection
+    #>
+
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory = $true)]
+        [ValidateNotNullOrEmpty()]
```

Comment:
You've used a `[ValidateScript()]` decoration for every other `-FilePath` parameter, but not this one.